documentation/public/user-guide/08-examples/03-jump-to-definition/index.md

```diff
@@ -0,0 +1,32 @@
+# 8.3. Jump to definition
+
+Another often used feature of an IDE is the ability to jump to the definition of a given identifier. Again, we can use the [Binding Graph API](../../07-semantic-analysis/02-binding-graph/index.md) to do it:
+
+```ts title="jump-to-definition.mts"
+--8<-- "documentation/public/user-guide/08-examples/03-jump-to-definition/examples/jump-to-definition.mts"
+```
+
+The following example shows jumping to the definition of the parameter `delta` in line 11:
+
+```solidity
```

Comment:
I suggest putting this in a separate code file, and asking `mkdocs` to render the line numbers instead:

https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-line-numbers

Reply:
.. same for others

Reply:
Heh, I was wondering if `mkdocs` had something like that. Apparently it can't highlight specific chunks of text, but I guess highlighting lines will do.